src/content/app/tools/blast/views/blast-submission-results/BlastSubmissionResults.tsx

```diff
@@ -167,35 +182,271 @@ type SingleBlastJobResultProps = {
   jobId: string;
   species: Species;
   diagramWidth: number;
+  blastDatabase: DatabaseType;
 };
 
+const hitsTableColumns: TableColumns = [
+  {
+    width: '0px',
+    columnId: 'id',
+    title: 'ID',
+    isFilterable: false,
+    isHideable: false
+  },
+  {
+    width: '130px',
+    columnId: 'e_value',
+    title: 'E-value',
+    isSortable: true,
+    helpText: (
+      <span>
+        The number of times a match is expected to occur by chance <br></br>{' '}
+        Typically, a low e-value indicated greater similarity between sequences{' '}
```

Comment:
What's the purpose of the empty strings here?

Reply:
`indicated`? or `indicates`?

Reply:
> What's the purpose of the empty strings here?

No idea were it is coming from!

Changed it to `indicates`